### ISSUE TITLE
Pretrained weights for CIFAR-10 Ablation

### ISSUE THREAD

**Issue by HashmatShadab** [MadryLab/smoothed-vit#3]
Hi

Thanks for sharing your work!

Can you please provide weights for ViT models trained on CIFAR-10?
Further, if possible can you add the clean and robust accuracy of the pre-trained models in the table as well for verifying the results?

**Comment by Hadisalman** [comment]
Hi @HashmatShadab! The CIFAR-10 models you requested can be found [here](https://drive.google.com/file/d/11IbEBBVszcA0jB08VESJwNA8HPtAYRMi/view?usp=share_link). For the clean and certified accuracy, please refer to the tables in the paper!

**Comment by HashmatShadab** [comment]
Thank you!
